Sync with ballerina 0.990.3 (#12)
wso2-ballerina/module-amazons3: README.md

import ballerina/io;
import wso2/amazons3;

amazons3:AmazonS3Configuration amazonS3Config = {
    accessKeyId: "<your_access_key_id>",
    secretAccessKey: "<your_secret_access_key>",
    region: "<your_region>",
    amazonHost: "<your_host_name>"
};

amazons3:Client amazonS3Client = new(amazonS3Config);

public function main(string... args) {

    string bucketName = "testBallerina";
    var createBucketResponse = amazonS3Client->createBucket(bucketName);
    if (createBucketResponse is amazons3:Status) {

>         // If successful, prints the status value as true.

Reviewer: ```suggestion
        // If successful, prints the status of the operation.
```

We can also remove the `status` variable and print inline.

Reviewer: Same for Module.md

>         // If successful, prints the status of the operation.

Reviewer: ```suggestion
        // If successful, print the status of the operation.
```
